pkg/worker/loader.go

```diff
@@ -98,10 +105,14 @@ func (loader *LoaderOSS) Build(ctx context.Context, model *arcadiav1alpha1.Typed
 		url = loader.endpoint.SchemeInternalURL()
 	}
 
+	img := defaultLoaderImage
+	if loader.worker.Spec.Loader.Image != "" {
+		img = loader.worker.Spec.Loader.Image
+	}
```

Comment:
set default pull policy too

Reply:
https://github.com/kubeagi/arcadia/pull/561/files#diff-8f0e3fc5aa037de327c89f467cafeabaf458c3026d56305ea04b3a7b5feb9c9fR30

Default pull policy is `IfNotPresent`